src/AcceptanceTests/Routing/When_sending_to_specific_namespace.cs

```diff
@@ -13,56 +11,38 @@ namespace NServiceBus.Azure.Transports.WindowsAzureServiceBus.AcceptanceTests.Ro
 
     public class When_using_multiple_namespaces : NServiceBusAcceptanceTest
     {
+        static string connectionString = connectionString = EnvironmentHelper.GetEnvironmentVariable("AzureServiceBusTransport.ConnectionString");
+        static string targetConnectionString = EnvironmentHelper.GetEnvironmentVariable("AzureServiceBus.ConnectionString.Fallback");
+
         [Test]
         public async Task Should_support_request_reply_across_namespaces_using_aliases()
         {
-            var runSettings = new RunSettings();
-            runSettings.TestExecutionTimeout = TimeSpan.FromMinutes(1);
-
-            var ctx = new AzureServiceBusTransportConfigContext();
-            ctx.Callback = (endpointName, extensions) =>
-            {
-                var connectionString = EnvironmentHelper.GetEnvironmentVariable("AzureServiceBusTransport.ConnectionString");
-                var targetConnectionString = EnvironmentHelper.GetEnvironmentVariable("AzureServiceBus.ConnectionString.Fallback");
-                extensions.UseNamespaceAliasesInsteadOfConnectionStrings();
-
-                if (endpointName == Conventions.EndpointNamingConvention(typeof(EndpointInTargetNamespace)))
-                {
-                    extensions.NamespaceRouting().AddNamespace("source", connectionString);
-                    extensions.NamespacePartitioning().AddNamespace("target", targetConnectionString);
-                }
-                else
-                {
-                    extensions.NamespacePartitioning().AddNamespace("source", connectionString);
-                    extensions.NamespaceRouting().AddNamespace("target", targetConnectionString);
-                }
-
-                var topology = EnvironmentHelper.GetEnvironmentVariable("AzureServiceBusTransport.Topology");
-                if (topology == "ForwardingTopology")
-                {
-                    extensions.UseForwardingTopology();
-                }
-                else
-                {
-                    extensions.UseEndpointOrientedTopology();
-                }
-            };
-
-            runSettings.Set("AzureServiceBus.AcceptanceTests.TransportConfigContext", ctx);
-
             var context = await Scenario.Define<Context>()
                 .WithEndpoint<EndpointInSourceNamespace>(b =>
                 {
-                    b.When(async (bus, c) =>
+                    b.CustomConfig(c =>
```

Comment:
Much cleaner,  ❤️  it.
Giving credit to @yvesgoeleven for the original creativity though 😃